app/views/layouts/angular/_multi_auth_credentials.html.haml

```diff
@@ -67,6 +67,12 @@
         .col-md-12{"ng-if" => "#{ng_model}" == "emsCommonModel" && "#{ng_model}.emstype == 'ec2'"}
           = render :partial => "layouts/angular-bootstrap/endpoints_for_url",
                                :locals => {:prefix => "default", :ng_model => "#{ng_model}"}
+          = render :partial => "layouts/angular-bootstrap/any_field",
+                               :locals => {:prefix     => "",
+                                           :input_name => "service_account",
+                                           :input_desc => "Assume role ARN",
```

Comment:
Missing gettext.

Reply:
fixed in https://github.com/ManageIQ/manageiq-ui-classic/pull/5621/commits/5705643011dc8d1552aaac094ef77a2b1c9e5f27

Reply:
the gettext call belongs here, not on the variable.

Reply:
The gettext needs to be applied here: `_("Assume role ARN")`